app/authentication/authenticator.py

```diff
@@ -160,14 +163,14 @@ def store_session(metadata):
     logger.info("user authenticated")
 
 
-def decrypt_token(encrypted_token):
+def decrypt_token(encrypted_token: str) -> Any:
     if not encrypted_token:
         raise NoTokenException("Please provide a token")
 
     logger.debug("decrypting token")
     decrypted_token = decrypt(
         token=encrypted_token,
-        key_store=current_app.eq["key_store"],
+        key_store=current_app.eq["key_store"],  # type: ignore
         key_purpose=KEY_PURPOSE_AUTHENTICATION,
         leeway=current_app.config["EQ_JWT_LEEWAY_IN_SECONDS"],
     )
```

Comment:
The type of decrypted token is a dict. When MyPy says expected some type but got `Any` that most times means the method you called doesn't have type hints, i.e decrypt in our case, but for those, you can specify the type inline. So this would become:

```py
def decrypt_token(encrypted_token: str) -> dict[str, Any]:
    if not encrypted_token:
        raise NoTokenException("Please provide a token")

    logger.debug("decrypting token")
    decrypted_token: dict[str, Any] = decrypt(
        token=encrypted_token,
        key_store=current_app.eq["key_store"],  # type: ignore
        key_purpose=KEY_PURPOSE_AUTHENTICATION,
        leeway=current_app.config["EQ_JWT_LEEWAY_IN_SECONDS"],
    )

    logger.debug("token decrypted")
    return decrypted_token
```